shopinvader/wizards/shopinvader_partner_binding_line.py

```diff
@@ -55,6 +55,12 @@ def action_apply(self):
             )
             if not partner_binding:
                 bind_values = {"backend_id": backend.id}
+                # Locomotive doesn't work with uppercase.
+                # And we have to do the write before the binding
+                email_lower = (record.partner_id.email or "").lower()
+                # Do the update only if necessary
+                if record.partner_id.email != email_lower:
+                    record.partner_id.write({"email": email_lower})
```

Comment:
I'd disable tracking
```suggestion
                    record.partner_id.with_context(tracking_disable=True).write({"email": email_lower})
```

Reply:
as you are writing again here, just define a dict of values and inject the email if needed